src/features/donations/components/PaymentRequestForm.tsx

```diff
@@ -3,9 +3,9 @@ import {
   useStripe,
 } from '@stripe/react-stripe-js';
 import { useEffect, useMemo, useState } from 'react';
-import AnimatedButton from '../../../common/InputTypes/AnimatedButton';
+import AnimatedButton from '../../common/InputTypes/AnimatedButton';
 import styles from './../styles/TreeDonation.module.scss';
-import i18next from '../../../../../i18n';
+import i18next from '../../../../i18n/server';
```

Comment:
Please be careful with this change - I think you must delete the `/server` again. This applies also the other files where you changed this.

Reply:
Yes, this is still in draft and experimental stage